File: test_admin_login.js
const axios = require('axios');

async function testLogin() {
    try {
        console.log('测试admin-api登录端点...');
        
        const response = await axios.post('http://localhost:11025/api/admin/auth/login', {
            username: 'admin',
            password: 'Abcdef1!'
        }, {
            headers: {
                'Content-Type': 'application/json',
                'Origin': 'http://localhost:13085'
            },
            validateStatus: function (status) {
                return true; // 允许任何状态码，以便我们可以查看错误
            }
        });
        
        console.log('状态码:', response.status);
        console.log('响应头:', response.headers);
        console.log('响应数据:', response.data);
        
    } catch (error) {
        console.error('错误:', error.message);
        if (error.response) {
            console.error('状态码:', error.response.status);
            console.error('响应数据:', error.response.data);
        }
    }
}

testLogin();

File: test_admin_login.py
import requests
import json

def test_login():
    try:
        print("测试admin-api登录端点...")
        
        response = requests.post(
            'http://localhost:11025/api/admin/auth/login',
            json={
                'username': 'admin',
                'password': 'Abcdef1!'
            },
            headers={
                'Content-Type': 'application/json',
                'Origin': 'http://localhost:13085'
            }
        )
        
        print(f"状态码: {response.status_code}")
        print(f"响应头: {dict(response.headers)}")
        print(f"响应数据: {response.text}")
        
    except Exception as e:
        print(f"错误: {str(e)}")

if __name__ == "__main__":
    test_login()